src/WebJobs.Script.Abstractions/WebJobs.Script.Abstractions.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

We shouldn't need multi-target semantics in a single TFM project
```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```

---

I believe this was in place to override what was defined in `common.props`, but common has been changed and no longer defines a target, so we should be OK to change (I think @liliankasem went through some of the projects to fix this up, but this was likely missed)